arch/risc-v/Kconfig

```diff
@@ -217,6 +217,11 @@ config ARCH_RV_ISA_C
 	bool
 	default n
 
+config ARCH_HAVE_LAZYFPU
+	bool
+	default y
```

Comment:
keep default to false, and select in ARCH_RISCV if ARCH_HAVE_FPU

Reply:
Yes I'll change on monday